js/main.js

```diff
@@ -43,23 +57,80 @@ $(document).ready(function () {
             }
         });
     });
-    //        probe 1:
-    //        $(".nav-item").each(function () {
-    //            var target = $(this).offset().top;
-    //            var id = $(this).attr('id');
-    //
-    //            if (position >= target) {
-    //                $("#main-nav ul li a").removeClass('activeMenu');
-    //
-    //                $("#main-nav ul li a[href=#]").addClass('activeMenu');
-    //            }
-    //        });
 
     $(".nav-item").click(function () {
         $(".nav-item").removeClass('activeMenu');
         $(this).addClass('activeMenu');
     });
 
+
```

Comment:
Hier sind ganz viele Funktionen definiert, die keine Verwendung mehr haben (Du nutzt nur noch fadeDown & fadeUp) - den kannst Du löschen. 
Auch hast Du in Deiner main.js sehr viele verschiedene $(window).scroll - Funktionen. Die können zu einer einzigen zusammengefasst werden. 

Statt noch zwei einzelne  Funktionen (fadeDown, fadeUp) zu haben, die im Grunde das gleiche machen, kann man das auch wie folgt vereinfachen: 

```
$.each(['.fadeDown', '.fadeUp'], function (i) {
            var bottomOfObject = $(this).position().top + $(this).outerHeight();
            var bottomOfWindow = $(window).scrollTop() + $(window).height();

            if (bottomOfWindow > bottomOfObject) {
                $(this).addClass('activate');
            }
        });
```

Wir suchen in dieser Funktion nach allen Elementen mit verwendeten Klassen fadeDown oder fadeUp. Wenn gefunden, schauen wir, ob der Nutzer schon in der richtigen Scroll-Position ist. Wenn ja, dann fügen wir eine Klasse "activate" hinzu. 

Was machen wir jetzt mit der Klasse "activate"?
Die können wir einfach für die jeweiligen Elemente definieren:

```
.fadeUp.activate {
    opacity: 0;
    animation-name: fadeInUp;
}

.fadeDown.activate {
    opacity: 0;
    animation-name: fadeInDown;
}
```

Damit haben wir das Gleiche erreicht wie zu vor, aber mit wesentlich weniger Code.